http/headers/Service-Worker-Allowed.json

```
@@ -0,0 +1,43 @@
+{
+  "http": {
+    "headers": {
+      "Service-Worker-Allowed": {
+        "__compat": {
+          "description": "`Service-Worker-Allowed` response header to broaden path restrictions for a service worker's default scope.",
+          "spec_url": "https://w3c.github.io/ServiceWorker/#service-worker-allowed",
```

MDN page is incoming shortly if we want to pre-empt this:

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Service-Worker-Allowed",
          "spec_url": "https://w3c.github.io/ServiceWorker/#service-worker-allowed",
```

---

It'll be automatically added, let's not add it for now.

---

oh, I guess the linter removes it if it doesn't exist